.github/workflows/test_component.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     runs-on: ${{ inputs.test_runs_on }}
     timeout-minutes: 210
     container:
-      image: ${{ inputs.platform == 'linux' && 'ghcr.io/rocm/no_rocm_image_ubuntu24_04@sha256:4150afe4759d14822f0e3f8930e1124f26e11f68b5c7b91ec9a02b20b1ebbb98' || null }}
+      image: ${{ inputs.platform == 'linux' && 'ghcr.io/rocm/no_rocm_image_ubuntu24_04:users-moberste-add_libhipcxx' || null }}
```

Comment:
Tagging so we don't forget - the dockerfile updates will need to be reviewed/merged as one PR/commit first, then this can pick up a newly built image. We can't merge this PR while the image points to a user image.